src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveFundingUpdates_20231106.s.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/GovV3Helpers.sol';
+import {EthereumScript} from 'aave-helpers/ScriptUtils.sol';
+import {AaveV3Ethereum_AaveFundingUpdates_20231106} from './AaveV3Ethereum_AaveFundingUpdates_20231106.sol';
+
+/**
+ * @dev Deploy Ethereum
+ * command: make deploy-ledger contract=src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveFundingUpdates_20231106.s.sol:DeployEthereum chain=mainnet
+ */
+contract DeployEthereum is EthereumScript {
+  function run() external broadcast {
+    // deploy payloads
+    AaveV3Ethereum_AaveFundingUpdates_20231106 payload0 = new AaveV3Ethereum_AaveFundingUpdates_20231106();
+
+    // compose action
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actions = new IPayloadsControllerCore.ExecutionAction[](1);
+    actions[0] = GovV3Helpers.buildAction(address(payload0));
+
+    // register action at payloadsController
+    GovV3Helpers.createPayload(actions);
+  }
+}
+
+/**
+ * @dev Create Proposal
+ * command: make deploy-ledger contract=src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveFundingUpdates_20231106.s.sol:CreateProposal chain=mainnet
+ */
+contract CreateProposal is EthereumScript {
+  function run() external {
+    // create payloads
+    PayloadsControllerUtils.Payload[] memory payloads = new PayloadsControllerUtils.Payload[](1);
+
+    // compose actions for validation
```

Comment:
As this pr was oppened quite a while ago, provably makes sense to update from main branch.
Then take a look at how the new proposals use the create proposals scripts, as it no longer needs to wait for payload creation to merge the AIP pr (as we use create2)

Reply:
You're right, very good point.